liquibase-standard/src/main/java/liquibase/database/Database.java

```diff
@@ -684,5 +685,9 @@ default void checkDatabaseConnection() throws DatabaseException {
     default String generateConnectCommandSuccessMessage() {
         return null;
     }
+
+    default String formatCommandForMdc(SqlStatement statement) {
+        return SqlUtil.getSqlString(statement, SqlGeneratorFactory.getInstance(), this);
+    }
```

Comment:
This should be called `formatStatementForMdc` and needs javadoc.

Reply:
done